modules/tinymce/CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Links in notification text did not show the correct mouse pointer #TINY-7661
 - The formatter match APIs were incorrectly returning false for formats that specified an attribute or style should be removed from an element #TINY-6149
 - The type signature on `editor.formatter.matchNode` had the wrong return type (was `boolean` but should have been `Formatter | undefined`) #TINY-6149
+- Using the Tab key to navigate into the editor on IE 11 would incorrectly focus the toolbar #TINY-3707
```

Comment:
Do we want (and is it easy) to add a test for this specifically for IE?

Reply:
The test would be a text input next to Tiny Editor, the user tabs from the input and it should focus on the editor, right?!

Reply:
Agreed we should add a test here... so yes, this would need to be a webdriver test with an element outside the editor that uses `RealKeys` to tab inside the editor IIUC.

Reply:
Test added.